src/components/SignupsSplit.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTreatments, SplitContext } from '@splitsoftware/splitio-react';
+
+const SignupsSplit = ({
```

Comment:
With the latest changes, this component becomes much more generally applicable. So let's rename it to something that doesn't relate to the test itself. Some ideas: `ABTester` or maybe `SplitTester`